python/pyspark/sql/dataframe.py

```diff
@@ -1393,6 +1393,16 @@ def replace(self, to_replace, value=None, subset=None):
         |null|  null| null|
         +----+------+-----+
 
+        >>> df4.na.replace('Alice', None).show()
```

Comment:
I guess it is `.na.replace` vs `.replace`. I think both should be the same though. I just built against this PR and double checked as below:

```python
>>> df = spark.createDataFrame([('Alice', 10, 80.0)]).replace("Alice")
```
```python
>>> df.replace("Alice").first()
```
```
Row(_1=None, _2=10, _3=80.0)
```
```python
>>> df.na.replace("Alice").first()
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: replace() takes at least 3 arguments (2 given)
```